openupgrade_scripts/scripts/website_slides/16.0.2.6/end-migration.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2025 Tecnativa - Carlos Lopez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+
+def _recompute_slide_slide_type(env):
```

Comment:
For having a trace of what's going on.

```suggestion
@openupgrade.logging()
def _recompute_slide_slide_type(env):
```

Reply:
Done